scripts/genproto.sh

```diff
@@ -6,64 +6,51 @@ set -e
 set -u
 
 if ! [[ "$0" =~ "scripts/genproto.sh" ]]; then
-	echo "must be run from repository root"
-	exit 255
+    echo "must be run from repository root"
+    exit 255
```

Comment:
Can you avoid the change from tab to spaces in this PR?  Makes it harder to review.